cli/jobs/single-step/gpu_perf/gpu_perf_job.yml

```diff
@@ -0,0 +1,46 @@
+# This runs a couple of tests on a gpu node.
+# It helps diagnosis of health of perf issues.
+
+type: command
+
+code: 
+  local_path: src
+
+command: /bin/bash ./gpu_perf.sh
+
+# to iterate quickly on your environment design, use inline
+environment: file:./environments/azureml/env.yml
+
+# to reuse with different parameters, use registered environment 
+# environment: azureml:nccltests_azureml:openmpi4.1.0-cuda11.1-cudnn8-ubuntu18.04
+# environment: azureml:nccltests_nvidia_pytorch:22.02-py3
+
+# NOTE: set env var if needed by your configuration
+environment_variables:
+  NCCL_DEBUG: "INFO" # adjusts the level of info from NCCL tests
+
+  # NCCL_IB_PCI_RELAXED_ORDERING: "1" # Relaxed Ordering can greatly help the performance of Infiniband networks in virtualized environments.
+  # NCCL_SOCKET_IFNAME: "eth0"
+  # NCCL_TOPO_FILE: "/opt/microsoft/ndv4-topo.xml"
+
+  # NCCL_IB_DISABLE: "1" # force disable infiniband (if set to "1")
+
+  # UCX_IB_PCI_RELAXED_ORDERING: "on"
+  # UCX_TLS: "tcp"
+  # UCX_NET_DEVICES: "eth0" # if you have Error: Failed to resolve UCX endpoint...
+
+  # CUDA_DEVICE_ORDER: "PCI_BUS_ID" # ordering of gpus
+
+
+compute: azureml:gpu-cluster
+
+distribution:
+  type: mpi
+  process_count_per_instance: 8 # NOTE: set equal to number of gpus on the node
+
+resources:
+  instance_count: 2 # NOTE: to use multiple nodes
+
+experiment_name: nccl-test
```

Comment:
there is a naming convention (+ order of the YAML) sections in the contributing guide, would be good to align

Reply:
same for display name

Reply:
I checked it and the yml follows the guidance provided, but for environment_variables and display_name there's no guidance.

here's that's in the guidelines https://github.com/Azure/azureml-examples/blob/main/cli/CONTRIBUTING.md#jobs
code > command > inputs > environment > compute > experiment_name > description